barretenberg/cpp/src/barretenberg/dsl/acir_format/avm_recursion_constraint.cpp

```diff
@@ -163,14 +167,6 @@ AggregationObjectIndices create_avm_recursion_constraints(Builder& builder,
         key_fields.emplace_back(field);
     }
 
-    // TODO(JEANMON): Once we integrate with public inputs, we will have to decide whether we inject (see
```

Comment:
was there an issue associated with this ?

Reply:
This comment is obsolete as we have already integrated the public inputs.